apps/website/lib/src/routes/index.dart

```diff
@@ -16,56 +18,83 @@ HTMLElement get index => basicLayout(
-        ),
-        HTMLParagraphElement()
-          ..style.fontSize = '2rem'
-          ..textContent = formatDate(event.date),
-        HTMLDListElement()
-          ..style.display = 'grid'
-          ..style.gridGap = '0.5rem'
-          ..style.gridTemplateColumns = 'repeat(2, 1fr)'
-          ..style.marginTop = '1rem'
-          ..style.fontSize = '1.2rem'
-          ..appendAll([
-            HTMLElement.dt()..textContent = text(contents.placeLabel),
-            HTMLElement.dd()..append(
-              externalLink(text(event.place.name), url: event.place.url)
-                ..style.fontSize = 'inherit',
-            ),
-          ]),
-        HTMLParagraphElement()
-          ..style.position = 'relative'
-          ..style.marginTop = '1rem'
-          ..appendAll([
-            externalLink(
-              text(event.blog.sponsorship.title),
-              url: text(event.blog.sponsorship.url),
-            ),
-            HTMLImageElement()
-              ..style.position = 'absolute'
-              ..style.left = '100%'
-              ..style.bottom = '-50%'
-              ..style.height = '5rem'
-              ..alt = 'FlutterKaigi Dash'
-              ..src = '/img/flutterkaigi_dash.png',
-          ]),
-        _countdown..style.marginTop = '1rem',
-      ]);
+HTMLElement get _mainContent {
+  final tagLineWords = event.tagLine.split(' ');
+
+  return HTMLElement.article()
+    ..style.display = 'flex'
+    ..style.flexDirection = 'column'
+    ..style.justifyContent = 'center'
+    ..style.alignItems = 'center'
+    ..appendAll([
+      HTMLDivElement()
+        ..style.position = 'relative'
+        ..style.whiteSpace = 'nowrap'
+        ..style.fontFamily = 'Lexend'
+        ..style.textAlign = 'center'
+        ..style.marginTop = '6rem'
+        ..style.padding = '6rem 8rem'
+        ..style.backgroundImage = 'url(/img/brush.webp)'
+        ..style.backgroundSize = 'contain'
+        ..style.backgroundRepeat = 'no-repeat'
+        ..appendAll([
+          for (var i = 0; i < tagLineWords.length; i++)
+            HTMLHeadingElement.h1()
+              ..textContent = tagLineWords[i]
+              ..style.display = 'block'
+              ..style.opacity = '0'
+              ..style.transform = 'translateX(100vw)'
+              ..style.fontStyle = 'italic'
+              ..style.fontSize = 'clamp(2.5rem, 6vw, 4.5rem)'
+              ..style.color = 'white'
+              ..style.animationDelay = '${i * 0.3}s'
+              ..style.animationName = 'slide-in'
+              ..style.animationDuration = '0.5s'
+              ..style.animationTimingFunction = _tagLineAnimationCurve
+              ..style.animationFillMode = 'forwards',
+        ]),
+      HTMLHeadingElement.h2()
+        ..textContent = event.title
+        ..style.fontSize = '2rem'
+        ..style.fontWeight = 'bold'
+        ..style.fontFamily = 'Lexend'
+        ..style.marginTop = '6rem',
+      HTMLHeadingElement.h3()
```

Comment:
場所とスポンサーリングについて，段落タグが良いと思うのですがどうでしょう
```suggestion
      HTMLParagraphElement()
```

Reply:
https://github.com/FlutterKaigi/2025/pull/116/commits/3bd6104dc8d3ba551cbf87e28abcc3184db7c791
修正しました！